.github/workflows/test.yml

```diff
@@ -85,28 +68,11 @@ jobs:
         choco install opencl-intel-cpu-runtime
         python -m pip install --only-binary=pyopencl --find-links http://www.silx.org/pub/wheelhouse/ --trusted-host www.silx.org pyopencl
 
-    - name: Fetch sources for sibling projects
+    - name: Install sibling projects
       run: |
-        git clone --depth=50 --branch=master https://github.com/SasView/sasmodels.git ../sasmodels
-        git clone --depth=50 --branch=master https://github.com/bumps/bumps.git ../bumps
-
-    - name: Build and install sasmodels
-      run: |
-        cd ../sasmodels
-        rm -rf build
-        rm -rf dist
-        python setup.py clean
-        python setup.py build
-        python -m pip install .
+        pip install git+https://github.com/bumps/bumps.git
```

Comment:
This would be nice if we were able to use `pip install git+https:..` everywhere and not just in this job. Can it also be used in `installers.yml` and `docs.yml`? I suspect not as those jobs need the unpacked sources for the `bumps` and `sasmodels` present to be able to build the `sasview` documentation.

The follow-on question is then: do we want to use different methods in different jobs?